templates/field/field--text-long--node.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
```

Comment:
as I can see same we can do for oe-contact--full.html.twig. where we can also add config for oe_contact entity type.